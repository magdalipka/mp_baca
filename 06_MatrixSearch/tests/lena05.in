1
6 7
0 0 1 2 3 4 4
1 2 2 3 3 4 4
2 2 4 4 4 5 6
3 3 4 4 4 6 6
7 7 7 7 7 7 8
9 9 9 9 10 11 11
8

