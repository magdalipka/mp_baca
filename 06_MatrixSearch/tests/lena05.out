RekPier: 8 w (4,6)
RekOst: 8 w (4,6)
IterPier: 8 w (4,6)
IterOst: 8 w (4,6)
---
